pkg/plugins/policies/meshtrace/plugin/v1alpha1/plugin.go

```diff
@@ -137,6 +145,52 @@ func applyToOutbounds(
 	return nil
 }
 
+func applyToRealResources(
+	ctx xds_context.Context,
+	rules core_rules.SingleItemRules,
+	rs *xds.ResourceSet,
+	proxy *xds.Proxy,
+) error {
+	for uri, resType := range rs.IndexByOrigin(xds.NonMeshExternalService) {
```

Comment:
Why not MES?

Reply:
Since we don't support policies on Egress, thought maybe no point to add it now